examples/cxxopts/v1.3.0

```diff
@@ -0,0 +1 @@
+./v1.x.x
```

Comment:
Unused?

Reply:
`v1.3.0` `v1.4.0` `v1.4.1` `v1.4.2` `v1.4.3` `v1.4.4` 
Above files are symnolic link and points to `v1.x.x` directory.
`v1.x.x` `v2.0.0` `v2.1.0` are directory that contain `example.cpp` source file.
`example.cpp` does not support lower versions. So, I divided into versions.

Reply:
Probably more clean solution will be to do:
```cmake
if(HUNTER_cxxopts_VERSION VERSION_LESS "2.0.0")
  set(source_dir "v1.x.x")
else()
  set(source_dir "v${HUNTER_cxxopts_VERSION}")
endif()

add_executable(foo "${source_dir}/example.cpp")
```